### ISSUE TITLE
Unable to reproduce accuracy

### ISSUE THREAD

**Issue by blue-sky-only** [hexiao-cs/CALNet-Dronevehicle#19]
Why did we end up with an accuracy 10 points lower than the paper using your configuration file.  The accuracy we obtained on the validation set after 50 epochs is 65.3 mAP (AP50). we use yolov5l_fusion_transformerx3_new_dronevehicle_aligned_cross.yaml, after train 50 epoch wtih RTX3090. The final accuracy is far from reaching 75mAP(AP50).

**Comment by jacksonwu09** [comment]
你好，我想问下，你在训练时没有出现问题吗，我出现了这样的问题，这个不是源代码吗，为啥还是会报错
![image](https://github.com/user-attachments/assets/e05c9a1d-3932-484d-8d3e-db67cb3d0251)

**Comment by Vehicle-AHU** [comment]
你好，我想问一下P，R，HBBmAP0.5,HBBmAP0.5:0.95论文中的结果是HBBmAP0.5吗？

**Comment by jacksonwu09** [comment]
> 你好，我想问一下P，R，HBBmAP0.5,HBBmAP0.5:0.95论文中的结果是HBBmAP0.5吗？

不是，需要使用工具，将HBB转换为OBB

**Comment by wzh326** [comment]
> > 你好，我想问一下P，R，HBBmAP0.5,HBBmAP0.5:0.95论文中的结果是HBBmAP0.5吗？
> 
> 不是，需要使用工具，将HBB转换为OBB

你好，我看作者在readme中给出的数据集label是HBB形式的，源代码中也没有发现针对OBB的处理，请问作者是怎样得出论文中的结果的呢？

**Comment by jacksonwu09** [comment]
> > > 你好，我想问一下P，R，HBBmAP0.5,HBBmAP0.5:0.95论文中的结果是HBBmAP0.5吗？
> > 
> > 
> > 不是，需要使用工具，将HBB转换为OBB
> 
> 你好，我看作者在readme中给出的数据集label是HBB形式的，源代码中也没有发现针对OBB的处理，请问作者是怎样得出论文中的结果的呢？

![image](https://github.com/user-attachments/assets/8db52e7d-69a8-4136-946b-8c185aa07630)
作者已经给出了转换的工具，在进行测试时，使用这种命令就可以将HBB转换为OBB

**Comment by Thomas-403** [comment]
> > > > 你好，我想问一下P，R，HBBmAP0.5,HBBmAP0.5:0.95论文中的结果是HBBmAP0.5吗？
> > > 
> > > 
> > > 不是，需要使用工具，将HBB转换为OBB
> > 
> > 
> > 你好，我看作者在readme中给出的数据集label是HBB形式的，源代码中也没有发现针对OBB的处理，请问作者是怎样得出论文中的结果的呢？
> 
> ![image](https://github.com/user-attachments/assets/8db52e7d-69a8-4136-946b-8c185aa07630) 作者已经给出了转换的工具，在进行测试时，使用这种命令就可以将HBB转换为OBB

请问一下大佬，所以说它这个仓库跑出来的实际上还是hbb的结果，然后再用这个脚本进行转换是嘛

**Comment by hexiao0275** [comment]
> > > > > 你好，我想问一下P，R，HBBmAP0.5,HBBmAP0.5:0.95论文中的结果是HBBmAP0.5吗？
> > > > 
> > > > 
> > > > 不是，需要使用工具，将HBB转换为OBB
> > > 
> > > 
> > > 你好，我看作者在readme中给出的数据集标签是HBB形式的，源代码中也没有找到针对OBB的处理，请问作者是怎样的论文中的结果呢？
> > 
> > 
> > ![image](https://github.com/user-attachments/assets/8db52e7d-69a8-4136-946b-8c185aa07630)作者已经给出了这种转换的工具，在进行测试时，使用就可以将HBB转换为OBB命令
> 
> 请问一下大佬，所以说这个仓库跑出来的效果还是hbb的结果，然后再用这个脚本进行转换是嘛

是的，当时是用DOTA的脚本转换的，参考ReadMe，HBB会很高超过82%mAP50
